src/third_party/koral/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+PROJECT(koral)
+cmake_minimum_required (VERSION 3.1)
+
+IF (${OpenMVG_USE_CUDA})
+  FIND_PACKAGE(CUDA REQUIRED)
+  FIND_PACKAGE(SSE REQUIRED)
+
+  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/config/)
+
+  SET (HEADERS CLATCH.h CUDALERP.h FeatureAngle.h KFAST.h KORAL.h Keypoint.h)
+
+  SET (SRC CLATCH.cu CUDALERP.cu)
+
+  LIST(APPEND CUDA_NVCC_FLAGS "-D_MWAITXINTRIN_H_INCLUDED -lineinfo -Xptxas -v -Xcompiler -fopenmp -use_fast_math -O3 -gencode arch=compute_30,code=sm_30 -gencode arch=compute_60,code=sm_60 --default-stream per-thread")
+
+  SET(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++11")
+
+  SET(CMAKE_CXX_STANDARD 11)
```

Comment:
CXX_STANDARD is already set by the master OpenMVG cmake
see [here](https://github.com/openMVG/openMVG/blob/develop_koral/src/CMakeLists.txt#L135)